CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Added
+- Add VideoTransformDevice interface stub
```

Comment:
I would phrase this as 

```
- Add `VideoTransformDevice` interface placeholder. This interface mirrors `AudioTransformDevice`. Choosing `VideoTransformDevice`s in `DefaultDeviceController` will be implemented in a future release.
```

and move it down below line 13 to contextualize with `AudioTransformDevice`. This is important guidance for builders, so it's worth being verbose.

Reply:
Sounds good.